doc/build/changelog/unreleased_20/7156.rst

```diff
@@ -0,0 +1,6 @@
+.. change::
+    :tags: postgresql, usecase
+    :tickets: 7156
+
+    Adds support for PostgreSQL MultiRange types, introduced
+    in PostgreSQL 14.
```

Comment:
**mike bayer** (zzzeek) wrote:

"Note that this feature is currently only tested with the ``psycopg`` driver and involves the use of the ``psycopg.extras.ranges`` extension module"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3728

Reply:
updated